metrics/code-commits.md

```diff
@@ -4,15 +4,56 @@
 What is the number of code commits?
```

Comment:
Maybe this is not really a part of the use case? I mean, the use case seems to be described in the file under use_cases, and this seems more like some redefinition of one of the metrics...